changelog.md

```diff
@@ -6,6 +6,8 @@
 
 * `cordformation`: The `sshd` port entry in `node` produces an entry in `node.conf` that matches the config API.
 
+* The Dockerform task exposes the sshdPort in docker-compose.yml
```

Comment:
This changelog entry is corrupt; not least because we're currently on 4.0.30-SNAPSHOT but also because there are two lines saying:

```
### Version 4.0.26
```